src/sentry/utils/snuba.py

```diff
@@ -904,9 +906,8 @@ def _raw_snql_query(
             )
             body = query.snuba()
 
-        with thread_hub.start_span(op="snuba_snql", description="run query") as span:
+        with thread_hub.start_span(op="snuba_snql.run", description=str(query)) as span:
             span.set_tag("snuba.referrer", referrer)
-            span.set_data("snql", str(query))
```

Comment:
I left the tag for "backwards compat", but removed this due to payload size limits